examples/config-cve.json

```diff
@@ -4,8 +4,8 @@
         "rootDirectory": "/tmp/zot"
     },
     "http": {
-        "address": "127.0.0.1",
-        "port": "8080"
+        "address": "0.0.0.0",
+        "port": "5000"
```

Comment:
why change this example? committed by mistake?

Reply:
Yes by mistake